model-optimizer/extensions/ops/gather.py

```diff
@@ -75,7 +77,7 @@ def infer(node: Node):
 
         # we import PermuteInputs locally because it uses Gather inside and we have recursive imports
         from mo.graph.perm_inputs import PermuteInputs
-        PermuteInputs().set_input_permutation(node.in_node(1), node, 'input:0', 'axis')
+        PermuteInputs().set_input_permutation(node.in_node(2), node, 'input:0', 'axis')
```

Comment:
how is this attribute affected by permutation if we have non-zero batch_dim?

Reply:
if `batch_dims` == 1 everything will be ok.
if `batch_dims` > 1 we will have big problems with permutation mechanism. Frankly speaking I don't know what will happen in such case but in most cases Gather received input with rank < 4 (we have communicated about this in our MO chat in messenger)

Reply:
Marked conversion resolved. Discussion is valuable but beyond of the scope of the current PR.